keps/sig-apps/20190318-Graduate-CronJob-to-Stable.md

```diff
@@ -0,0 +1,577 @@
+---
+title: graduate-cronjob-to-stable  
+authors:  
+  - "@barney-s"  
+
+owning-sig: sig-apps  
+participating-sigs:
+  - sig-scalability
```

Comment:
This is the first time I see this KEP - if SIG-scalability is supposed to be participating SIG, I think that someone from the SIG should be in approvers list.

I'm happy to be that person - I just very briefly skimmed through the KEP - will add more comments later this week (apparently it won't hit the 1.18 timeframe given today is feature freeze).

Reply:
done.